pipelines/cemba/cemba_methylcseq/README.md

```diff
@@ -1,40 +1,184 @@
-  <li>Methylpy mpileup and allc tables
-  <li>Create allc file (allc_SAMPLE.tsv.bgz) and corresponding tabix index (.tbi)
-  <li>Create binc files, genebody, promoter (Will be available in next methylpy release)
-</ol>
-
-## Submitting Workflows to CaaS
-
-To run workflows in the cloud, first generate an options file containing CaaS secrets.
-*DO NOT* check the generated file in.
-```bash
-$ ./scripts/inject-caas-secrets <path/to/options.json> > <path/not/in/repo/options.caas.json>
-```
-
-Then submit your workflow with the generated options in one of two ways:
-
-### Manually via Swagger
-
-Follow the instructions [here](https://docs.google.com/document/d/1od3J2IgO5z27BKh8DEP4lbPilDE1t6f4xk8DdvkxGmI/edit#).
-When told to upload options JSON, choose to upload the options file generated by `inject-caas-secrets`.
-
-### Via the CLI
-
-TODO after the [`cromwell-tools` refactor](https://github.com/broadinstitute/cromwell-tools/pull/43) exposes CaaS-related
-options on the CLI.
+| Pipeline Version | Date Updated | Documentation Author | Questions or Feedback |
+| :----: | :---: | :----: | :--------------: |
+| [CEMBA_v1.0](CEMBA.wdl) | July 28, 2020 | [Elizabeth Kiernan](mailto:ekiernan@broadinstitute.org) | Please file GitHub issues in warp or contact [Kylee Degatano](mailto:kdegatano@broadinstitute.org) |
+
+# Table of Contents
+
+- [CEMBA Pipeline Overview](#cemba-pipeline-overview)
+  * [Introduction to the CEMBA Workflow](#introduction-to-the-cemba-workflow)
+  * [Quick Start Table](#quick-start-table)
+- [Set-up](#set-up)
+  * [CEMBA Installation and Requirements](#cemba-installation-and-requirements)
+  * [Inputs](#inputs)
+    + [Sample data input](#sample-data-input)
+    + [Additional Inputs](#additional-inputs)
+- [CEMBA Tasks and Tools](#cemba-tasks-and-tools)
+  * [CEMBA Task Summary](#cemba-task-summary)
+- [Outputs](#outputs)
+- [Versioning](#versioning)
+- [Have Suggestions?](#have-suggestions)
+
+
+
+# CEMBA Pipeline Overview
+
+<img src="CEMBA.png" width="750">
+
+## Introduction to the CEMBA Workflow
+
+CEMBA is a pipeline developed by the Data Coordination Platform (DCP) of the [Human Cell Atlas (HCA) Project](https://data.humancellatlas.org/) that supports processing of multiplexed single-nuclei bisulfite sequencing data. It is an alignment and methylated base calling pipeline that trims adaptors, attaches cell barcodes, aligns reads to the genome, filters reads based on quality and creates a VCF with methylation-site coverage. 
+
+*Interested in using the pipeline for your publication? See the [“CEMBA publication methods”](CEMBA.methods.md) for a generic "methods" style description of the pipeline.*
+
+
+## Quick Start Table
+
+| Pipeline Features | Description | Source |
+|-------------------|---------------------------------------------------------------|-----------------------|
+| Assay Type | Single-nucleus methylcytosine bisulfite sequencing | [Luo et al. 2017](https://science.sciencemag.org/content/357/6351/600/tab-pdf)
```

Comment:
Does this need to be updated to what Hanqing just shared? (I didn't click link but I thought he cited something from 2018)

Reply:
It's 2017- I updated the Methods paragraph doc.

Reply:
great thanks!